syntax = "proto3";

package arcticc.pb2.azure_storage_pb2;

message Config {
    string container_name = 1;
    string credential_name = 2;
    string credential_key = 3;
    string endpoint = 4;
    uint32 max_connections = 5;
    uint32 connect_timeout = 6;
    uint32 request_timeout = 7;
    bool ssl = 8;
    string prefix = 9;
    bool https = 10;
    bool connect_to_azurite = 11;
}
